M2/Macaulay2/d/M2mem.c

```diff
@@ -80,7 +80,7 @@ void freemem(void *s) {
 #ifdef MEMDEBUG
      M2_debug_free(s);
 #else     
-     GC_FREE(s);
+     /* GC_FREE(s); */
```

Comment:
I'm a little confused. Doesn't this make `freemem` and `freememlen` entirely useless? Also, keeping `trapchk` and `M2_debug_free` there would mean when compiling in memory debug mode you get in traps in places that normally wouldn't call `GC_FREE` anymore, so shouldn't those be commented too?

Reply:
Right, this makes `freemem()`/etc. into noops. But Mike has talked about removing gc from the engine, so you might change your mind later, and want the `freemem()` calls to still exist. Also, I don't really know how Macaulay2 debugging works or is used. I tried to note in a TODO in this github discussion that someone should ideally look at all the calls to `GC_FREE()` and decide what to do. But you certainly want to comment out the major calls to `GC_FREE()` at least, and it helps (a lot with parallelism) not hurts, so I did that.